attributes/security.rb

```diff
@@ -65,6 +65,18 @@
     end
   end
 
+  # Add cdap group to core-site.xml's hadoop.proxyuser.hive.groups
+  if node['hadoop'].key?('core_site') && node['hadoop']['core_site'].key?('hadoop.proxyuser.hive.groups')
+    arr = node['hadoop']['core_site']['hadoop.proxyuser.hive.groups'].split(',')
+    group = 'cdap'
+    unless arr.include?(group)
+      arr += [group]
+      default['hadoop']['core_site']['hadoop.proxyuser.hive.groups'] = arr.join(',')
+    end
+  else
+    default['hadoop']['core_site']['hadoop.proxyuser.hive.groups'] = 'cdap,hadoop'
```

Comment:
This is groups, not users. The `hive` user is a member of `hadoop` just like the other service accounts.